bytecomp/matching.ml

```diff
@@ -1360,18 +1360,50 @@ let val_of_option_bs_primitive : Lambda.primitive =
       prim_native_name = "";
       prim_native_float = false
     }
-    
+
+let val_of_unbox_option_bs_primitive : Lambda.primitive =
```

Comment:
This is called `val_of_unbox_option...` but the primitive is called `val_unbox_from...` can they have more consistent naming?